src/components/frame/RouteLayouts/UserNavigation/UserNavigation.tsx

```diff
@@ -75,21 +85,21 @@ const UserNavigation = () => {
           </div>
         </Tooltip>
       )}
-      {/*
-        <div className={`${styles.elementWrapper} ${styles.walletWrapper}`}>
-          {canInteractWithNetwork && nativeToken && userLock && (
-            <UserTokenActivationButton
-              nativeToken={nativeToken}
-              userLock={userLock}
-              colony={colonyData?.processedColony}
-              walletAddress={walletAddress}
-              dataTest="tokenActivationButton"
-            />
-          )}
-        </div>
-      */}
+      <div className={`${styles.elementWrapper} ${styles.walletWrapper}`}>
+        {/* {canInteractWithNetwork && colony?.nativeToken && userLock && ( */}
```

Comment:
Is there still the concept of userLock?

Reply:
What was this referring to? The staked/locked tokens? Or something else?

Reply:
@rdig Im not sure what its from, I just saw userLock on userdata from the dapp and didn't know if that should be ported

Reply:
I am going to remove this comment for now